core/types/rlp_payload.libevm.go

```diff
@@ -87,11 +96,14 @@ func TestOnlyClearRegisteredExtras() {
 var registeredExtras register.AtMostOnce[*extraConstructors]
 
 type extraConstructors struct {
-	stateAccountType           string
-	newHeader, newStateAccount func() *pseudo.Type
-	cloneStateAccount          func(*StateAccountExtra) *StateAccountExtra
-	hooks                      interface {
+	stateAccountType  string
+	newHeader         func() *pseudo.Type
+	newBody           func() *pseudo.Type
+	newStateAccount   func() *pseudo.Type
```

Comment:
```suggestion
	newHeader, newBody, newStateAccount         func() *pseudo.Type
```
These do the same thing so it's useful to signal that to the reader.

Reply:
I split them up since it was going pretty long and hard to read, especially if we later add more i.e. `newBlock`. I would suggest to keep them on separate lines, but I can revert to a single line if you feel strongly about this.